scripts/utils/price-utils.js

```diff
@@ -139,9 +157,44 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return true
   }
 
+  const checkNoProfitableOffer = async (order, exchange, globalPriceStorage = null) => {
+    const tokenInfo = fetchTokenInfoFromExchange(exchange, [order.buyToken, order.sellToken])
+    const currentMarketPrice = await getDexagPrice(
+      (await tokenInfo[order.buyToken]).symbol,
+      (await tokenInfo[order.sellToken]).symbol,
+      globalPriceStorage
+    )
+
+    // checks whether the order amount is negligible
+    if ((await valueInUSD(order, tokenInfo, globalPriceStorage)) < 1) {
```

Comment:
Here we are comparing a string with an integer. I'm not sure what the result is in general, but I'd cast the string to integer to be safe.